src/app/home/home.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { QuestionService } from '../shared/services/question.service';
+import { Question } from '../shared/models/question.model';
+import { Topic } from '../shared/models/topic.model';
+import { TopicService } from '../shared/services/topic.service';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss']
+})
+export class HomeComponent implements OnInit {
+  topics : Topic[];
+  last_questions : Question[];
+
+  constructor(private topicService: TopicService,
+    private questionService: QuestionService) { }
+
+  ngOnInit(): void {
+    // Setting page background color
+    document.body.style.backgroundColor = '#f1f2f4';
```

Comment:
Why can't you do it in scss file?

Reply:
You cannot directly set <body> properties through style file because of Angular view encapsulation.